.github/settings.yml

```diff
@@ -160,7 +182,11 @@ labels:
 - name: dev/wont-fix
```

Comment:
when would I use `dev/wont-fix`?

Reply:
As a reviewer to tell the PR-creator that his contribution won't fix the issue? 
I don't know.. but I don't see the necessity of this label as well^^

Reply:
when you close an issue and decide to NOT fix/implement it - maybe after a longer discussion, where you've come to that decision

Reply:
I see the usecase but not the benefit. If a PR is closed without merging I would implicitly assume that the PR won't fix the problem. Either way I would open the PR and check the discussion

Reply:
It's when an issue is something that is either expected or when there is something that it needs but cannot be done by you, like if the problem is in an external library. You still understand that it's a problem but you don't have any time, or way of fixing it.